### PR TITLE
performance_test: 1.2.1-1 in 'foxy/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by revati-naik**
Increasing version of package(s) in repository `performance_test` to `1.2.1-1`:

- upstream repository: https://gitlab.com/ApexAI/performance_test.git
- release repository: https://github.com/ros2-gbp/performance_test-release.git
- distro file: `foxy/distribution.yaml`
- bloom version: `0.9.1`
- previous version for package: `1.0.0-1`

### CODE REVIEW

**Comment by adityapande-1995** [comment]
Holding for [Foxy sync](https://discourse.ros.org/t/preparing-for-foxy-sync-2022-07-01/26297)

File: foxy/distribution.yaml
Comment (nuclearsandwich) [inline foxy/distribution.yaml:3416]: Now that this is out of sync hold I think it is blocked on the same comment as the other distributions. https://github.com/ros/rosdistro/pull/33763#discussion_r916281228

**Review by nuclearsandwich — CHANGES_REQUESTED** [review]
The doc entry requires an update before this is ready to merge.

**Review by tfoote — APPROVED** [review]
@revati-naik Please update the docs to be on a branch.

**Pull request merged**
